turbo.json

```diff
@@ -25,7 +25,8 @@
     "VERCEL",
     "VITE_CLERK_*",
     "EXPO_PUBLIC_CLERK_*",
-    "REACT_APP_CLERK_*"
+    "REACT_APP_CLERK_*",
+    "MODE"
```

Comment:
This is added because we've added a global env var (`import.meta.env.MODE`) in the server SDK Metadata

Reply:
🙃 Is it possible to somehow bypass this with an eslint disable rule ? I'd avoid adding this here, because it's not clear what it does, unless we add an inline comment inside `turbo.json`

Reply:
Yes it's possible to bypass this. Now we have 2 instances of `import.meta.env.MODE`, one is inside integration. I agree to bypass it for now, it's unclear what it does. Are we okay with bypassing for now @panteliselef ?

Reply:
Updated